src/style.css

```diff
@@ -1 +1,64 @@
 /* Styles go here */
+body {
```

Comment:
I would probably provide more generous CSS styles reset like:
```
*, *::after, *::before {
  box-sizing: border-box;
  margin: 0;
  padding: 0;
}
```
but that's a good start 😄 

[more about custom CSS resets](https://www.joshwcomeau.com/css/custom-css-reset/)